src/ToricVarieties/LineBundles/constructors.jl

```diff
@@ -23,7 +23,7 @@ Construct the line bundle on the abstract normal toric variety `v` with class `c
 """
 function ToricLineBundle(v::AbstractNormalToricVariety, input_class::Vector{fmpz})
     class = picard_group(v)(input_class)
-    return ToricLineBundle(v, class)
+    return ToricLineBundle(v, class, Dict())
```

Comment:
I think it would be easier to add a default 2-argument constructor, see the "note" in `https://nemocas.github.io/AbstractAlgebra.jl/dev/misc/#AbstractAlgebra.@attributes`.

Reply:
I am confused. The above is a 2-argument constructor. What is wrong with that?

Reply:
Sorry, I mean that if you add a default inner constructor in line 9, then you can just do `ToricLineBundle(v, class)` here. See the example in the note.